Islandora-CLAW/islandora
# Check derivative term

**GitHub Issue**: (link)
https://github.com/Islandora/documentation/issues/1475

* Other Relevant Links (Google Groups discussion, related pull requests,
 Release pull requests, etc.)

# What does this Pull Request do?

Fixes a check for `$derivative_term` after it has been fetched by URI.

# What's new?
Should trap cases where a derivative term can't be located by URI.

# How should this be tested?

* Add a Repository Item with Model=Image
* Add media, type=Image, Use=Original Term
* When media is submitted, and derivatives generated, you might see an error message "Error generating event: Could not locate derivative term with urihttps://projects.iq.harvard.edu/fits"

# Interested parties
@Islandora-CLAW/committers
